riscv-overlay-software-standard-draft.adoc

```diff
@@ -100,97 +99,96 @@ The implementation will be based on the RISC-V ISA and RISC-V tools but can be a
 
 The following sections specify the requirements for:
 
-1.  Run Time Engine – SW module which is responsible for managing the overlay heap. This module is designated to be part of operational SW
-2.  Toolchain – broad collection of programming tools (e.g. compiler, linker, debugger, etc.) which needs to develop SW applications.
+1.  RT-Engine – SW module which is responsible for managing the overlay heap. This module is designated to be part of operational SW.
+2.  Toolchain – broad collection of programming tools (e.g. compiler, linker, debugger, etc.) needed to develop SW applications.
 
 [[run-time-engine]]
-=== Run Time Engine
+=== RT-Engine
 
 [[general]]
 ==== General
 
-1.  Since SW can be more flexible then HW, we should not use direct mapping approach for overlay. For a small allocated overlay heap we can mapped any amount of code.
-2.  Functions and const-data can be in overlay
-3.  Functions/Const-data will be encapsulate to `groups`, overlay group.
-4.  Engine will be aware of the functions in the group, and will be aware how to address them.
-5.  Engine will manage load/evict of groups by providing hooks to be impemented by platform.
-6.  Can run on a bare metal system or under RTOS. Engine should be aware of RTOS usage so it must be thread safe – any given thread can invoke overlay functions.
+1.  Since SW can be more flexible then HW, we should not use a direct mapping approach for overlays. For a small allocated overlay heap we can map any amount of code (within limits).
```

Comment:
I think we should remove the ```within limits``` since the limits are due to bit space. in 32bit machine, we will have a limited token but maybe we could do "double-token" ?! ... my point here is that we don't have a real limitation

Reply:
We cannot map in code that is larger than the small allocated overlay heap. "Within limits" could be clearer though e.g. "we can any amount of code that will fit"

Reply:
Yes you can. you can map XGB to a few Kbytes of heap. The limitation is in the token = how much it can index.